x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -337,4 +338,32 @@ export function registerIndexRoutes({
       });
     })
   );
+
+  router.delete(
```

Comment:
Just a note - you'll also want to add a test for the route as well. With this [backend PR merged yesterday](https://github.com/elastic/kibana/pull/140459), I [started a test file](https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.test.ts) for the routes under `indices` where we can add these.

(I'm going to add a new issue as well as a stretch goal to move the ML-specific routes out of here and to their own file - we can probably do this after FF)

Reply:
Test added.